attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
                   [Room.current.id]
                 else
+                  if DRStats.necromancer? and @force_power_walk
```

Comment:
You've already checked for a force power walk by the time you've gotten in here, this check is redundant.